### ISSUE TITLE
Tabs sometimes fail to automatically mute in Chrome 54

### ISSUE THREAD

**Issue by ivan** [ludios/mute-new-tabs#4]
Possibly related to seeing this error:

```
Unchecked runtime.lastError while running tabs.update: No tab with id: 625.
```

```
function muteTab(tabId) {
    assert(Number.isInteger(tabId));
    chrome.tabs.update(tabId, {muted: true}); // <------
}
```

**Comment by ivan** [comment]
This seems to happen on cnn.com frequently

**Comment by ivan** [comment]
Probably related to #1 - tab IDs are being recycled, and Mute New Tabs thinks the tab is still on the same origin.  Something else may also be going on, e.g. tabs being left around by Chrome, even after being closed, but I'm not sure.

**Comment by ivan** [comment]
Probably fixed in the 1.0.0 release pending approval in the Chrome Web Store
